x-pack/plugins/ml/common/constants/alerts.ts

```diff
@@ -20,8 +21,26 @@ export const TOP_N_BUCKETS_COUNT = 1;
 
 export const ALL_JOBS_SELECTION = '*';
 
-export const HEALTH_CHECK_NAMES = {
+export const HEALTH_CHECK_NAMES: Record<JobsHealthTests, string> = {
   datafeed: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.datafeedCheckName', {
     defaultMessage: 'Datafeed is not started',
   }),
+  mml: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.mmlCheckName', {
+    defaultMessage: 'Model memory limit',
```

Comment:
Shouldn't this message be a statement like the rest of the messages below? For example, something like 'Job has reached the model memory limit.'

Reply:
Agree with @szabosteve. `Job has reached the model memory limit` or just `Model memory limit reached` ?

![image](https://user-images.githubusercontent.com/7405507/127140742-56a3be14-7570-4365-bd47-c6fb25bef274.png)

Reply:
Changed in b73d27b7abebb21dd393529cca351d0af5eb43bc